MekHQ/data/universe/awards/standard.xml

```diff
@@ -0,0 +1,552 @@
+		<size>Battalion</size>
+		<range>Mission</range>
+	</award>
+	<award>
+		<name>Distinguished Service</name>
+		<description>12 pilot kills in a mission.</description>
+		<medal>DistinguishedServiceM.png</medal>
+		<ribbon>1-03-4-DistinguishedService.png</ribbon>
+		<xp>3</xp>
+		<stackable>true</stackable>
+		<position>1.03.4</position>
+		<qty>12</qty>
+		<item>Kills</item>
+		<size>Individual</size>
+		<range>Mission</range>
+	</award>
+	<award>
+		<name>Silver Star</name>
+		<description>8 pilot kills in a scenario.</description>
+		<medal>SilverStarM.png</medal>
+		<ribbon>1-04-1-SilverStar.png</ribbon>
+		<xp>4</xp>
+		<stackable>true</stackable>
+		<position>1.04.1</position>
+		<qty>8</qty>
+		<item>Kills</item>
+		<size>Individual</size>
+		<range>Scenario</range>
+	</award>
+	<award>
+		<name>House Superior Service</name>
+		<description>144 company kills in a mission.</description>
+		<medal>HouseSuperiorServiceM.png</medal>
+		<ribbon>1-05-1-HouseSuperiorService.png</ribbon>
+		<xp>4</xp>
+		<stackable>true</stackable>
+		<position>1.05.1</position>
+		<qty>144</qty>
+		<item>Kills</item>
+		<size>Company</size>
+		<range>Mission</range>
+	</award>
+	<award>
+		<name>Legion of Merit</name>
+		<description>48 lance kills in a mission.</description>
+		<medal>LegionOfMeritM.png</medal>
+		<ribbon>1-05-2-LegionOfMerit.png</ribbon>
+		<xp>3</xp>
+		<stackable>true</stackable>
+		<position>1.05.2</position>
+		<qty>48</qty>
+		<item>Kills</item>
+		<size>Lance</size>
+		<range>Mission</range>
+	</award>
+	<award>
+		<name>Armed Forces</name>
+		<description>324 battalion kills in a mission.</description>
+		<medal>ArmedForcesM.png</medal>
+		<ribbon>1-07-2-ArmedForces.png</ribbon>
+		<xp>4</xp>
+		<stackable>true</stackable>
+		<position>1.07.2</position>
+		<qty>324</qty>
+		<item>Kills</item>
+		<size>Battalion</size>
+		<range>Mission</range>
+	</award>
+	<award>
+		<name>Bronze Star</name>
+		<description>4 pilot kills in a scenario.</description>
+		<medal>BronzeStarM.png</medal>
+		<ribbon>1-08-1-BronzeStar.png</ribbon>
+		<xp>3</xp>
+		<stackable>true</stackable>
+		<position>1.08.1</position>
+		<qty>4</qty>
+		<item>Kills</item>
+		<size>Individual</size>
+		<range>Scenario</range>
+	</award>
+	<award>
+		<name>Purple Heart</name>
+		<description>Wounded in combat due to enemy fire.</description>
+		<medal>PurpleHeartM.png</medal>
+		<ribbon>1-09-1-PurpleHeart.png</ribbon>
+		<stackable>true</stackable>
+		<position>1.09.1</position>
+		<qty>1</qty>
+		<item>Wounded</item>
+		<size>Individual</size>
+		<range>Event</range>
+	</award>
+	<award>
+		<name>House Meritorious Service</name>
+		<description>108 company kills in a mission.</description>
+		<medal>HouseMeritoriousServiceM.png</medal>
+		<ribbon>1-10-1-HouseMeritoriousService.png</ribbon>
+		<xp>3</xp>
+		<stackable>true</stackable>
+		<position>1.10.1</position>
+		<qty>108</qty>
+		<item>Kills</item>
+		<size>Company</size>
+		<range>Mission</range>
+	</award>
```

Comment:
Why was this completely changed from the original's 144 company kills in a mission?